Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/attendee/checkins/CheckInsFragment.kt

```diff
@@ -58,6 +61,19 @@ class CheckInsFragment : Fragment(R.layout.trace_location_attendee_checkins_frag
             adapter = checkInsAdapter
             addItemDecoration(TopBottomPaddingDecorator(topPadding = R.dimen.spacing_tiny))
             itemAnimator = DefaultItemAnimator()
+            onScroll { extend ->
+                if (extend) binding.scanCheckinQrcodeFab.extend() else binding.scanCheckinQrcodeFab.shrink()
```

Comment:
The collapse QR code icon is off-center, likely because the original icon is off-center, I did that a bit workaroundish to make it look correctly, I had trouble getting the right asset exported, can you fix it for me?

![Screenshot from 2021-03-25 10-15-16](https://user-images.githubusercontent.com/1439229/112449733-03799100-8d54-11eb-8eb8-9c43a5a0c95d.png)

Reply:
Fixed along with changing the icon in BottomNav for consistency
![device-2021-03-25-121350](https://user-images.githubusercontent.com/25054729/112464612-febcd900-8d63-11eb-928b-b6d81592d6ab.png)